cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -115,7 +116,7 @@ public static void validateConfig(SecureStore secureStore, SourceControlConfig s
     try {
       credentialsProvider.refresh();
     } catch (AuthenticationConfigException e) {
-      throw new RepositoryConfigValidationException("Failed to get authentication credentials: " + e.getMessage(), e);
+      throw new RemoteRepositoryValidationException("Failed to get authentication credentials: " + e.getMessage(), e);
```

Comment:
If the credentials aren't present in Secure store, then shouldn't it be a 400 error? An IOException while reaching out to secure store service will be a server error, but the user should ensure the secure store password is present.

Reply:
Changed back to throw`RepositoryConfigValidationException`